response_operations_ui/views/messages.py

```diff
@@ -181,15 +182,22 @@ def view_selected_survey(selected_survey):
 
         is_closed = request.args.get('is_closed', default='false')
 
+        thread_count = message_controllers.get_conversation_count({'survey': survey_id,
+                                                                   'is_closed': is_closed})
+
+        recalculated_page = _calculate_page(page, limit, thread_count)
```

Comment:
minor comment :- maybe name var as recalculated_page_num . On first scan its not clear if _calculate_page returns a page object of some kind or number